tests/e2e/specs/shopper/mini-cart.test.js

```diff
@@ -559,4 +562,77 @@ describe( 'Shopper → Mini Cart', () => {
 			await expect( page ).toMatch( productTitle );
 		} );
 	} );
+
+	describe( 'Shopper → Mini Cart → Translations', () => {
+		beforeAll( async () => {
+			await merchant.changeLanguage( 'nl_NL' );
+			await shopper.block.emptyCart();
+		} );
+
+		afterAll( async () => {
+			await merchant.changeLanguage( '' );
```

Comment:
I'm not 100% sure this will revert back to english, best to be specific here an chage to `'en_EN'`.